src/main/scala/leon/grammars/ValueGrammar.scala

```diff
@@ -25,6 +25,13 @@ case object ValueGrammar extends ExpressionGrammar[TypeTree] {
         terminal(InfiniteIntegerLiteral(1)),
         terminal(InfiniteIntegerLiteral(5))
       )
+    case StringType =>
+      List(
+        terminal(StringLiteral("")),
+        terminal(StringLiteral("a")),
+        terminal(StringLiteral("\"'\n\r\t")),
```

Comment:
For this one, I moved this special string evaluation to the disambiguation section, so that this one uses `"foo"` and `"bar"`.